### DISCUSSION TITLE
No Live Signal from webOS PicCap Grabber

### DISCUSSION THREAD

**Discussion by NatyaSadella** [awawa-dev/HyperHDR#436] [Q&A]
Hi!

I would like to finish my ambilight setup but I can't get a picture when clicking on LIVE PICTURE. I can control the lights with hyperhdr and PicCap says it's capturing video and ui. How to see the little Live Picture?

**Comment by awawa-dev** [comment]
Hi
webos is not supported here. You must check HyperHDR logs to determine if something is trying to connect using flatbuffers.
